# General Comments

* In intro there is a sentence with 21 strains followed by 3 genomes! It is not very clear!
* sub-MIC? what does it stand for? 
* What is the significance with *Haemophilus influenzae*?

# Main questions

* **Do you actually see a significant improvement in searching for human reads with Bowtie2 after Kraken2? And what is a second Kraken2 report?**
* **Did you implement rRNA removal?**
* what is the difference between core, pan and soft-core genomes?
* **Something like `OrthoFinder` would that be better than `Roary` especially for `PA7`?**
* Reannotation with Diamond for `group_????` explanation!?
* Did you try PANTHER for all genes or DE genes?
* `The percentage of unclassified genes in each total number of genes in the 3 different genomes was 59.9% for pangenome, 53.6% for core and 68.2% and for soft-core genomes`.
* Figure 3, Not sure what it actually shows! what is the significance?
* **How is the sputum collected and is it expected to have a lot of human reads compared to let's say fecal samples or oral samples?**
* Volcano plot missing! Also LFC > 1, what is control: sputum or cultures?
* How does DGE from PA01 compare to core/pangenome? 
* **Can you think any other way apart from pangenome to detect developing antibiotic resistance?** 
* Github, Workflow-Manager? 